README.md

```diff
@@ -1,17 +1,72 @@
 # Overview
 
-Common repo for Apiary metastore application automation, contains Terraform logic to stand up Apiary metastore.
+ This repo contains a terraform module to deploy datalake component in Apiary project, module deploys various stateful components in a typical hadoop compatible datalake in AWS.
```

Comment:
There are quite a few grammar and other small things I'd like to change but it would be a lot faster if I just edited the file directly rather than putting lots of comments. @rpoluri are you OK with me doing that?

Reply:
@massdosage , yes you can edit file directly. Thanks!